torch/__init__.py

```diff
@@ -948,7 +948,7 @@ def _check_with(error_type, cond: Union[builtins.bool, SymBool], message: Callab
     if not isinstance(cond, (builtins.bool, torch.SymBool)):
         raise TypeError(f'cond must be a bool, but got {type(cond)}')
 
-    if cond:
+    if torch.fx.experimental.symbolic_shapes.expect_true(cond):
```

Comment:
nb. You can do a horrible, horrible thing, and locally overwrite the meaning of `<` via a macro so that you automagically go from using `operator<` to `lt`.

Reply:
Also, this line is fantastic. Gotta love having just one entrypoint for a given thing.